### ISSUE TITLE
Where to report "fraudulent" package squatting?

### ISSUE THREAD

**Issue by asottile** [pypa/warehouse#4629]
I'd like to report a user for squatting on package names but don't see an avenue for doing so -- is there a mechanism for this?

perhaps as part of PEP541?

**Comment by pradyunsg** [comment]
It's being tracked in #1506.

We basically need to figure out the process here: https://github.com/pypa/warehouse/issues/1506#issuecomment-375736028.

**Comment by asottile** [comment]
Does PEP 541 cover malicious squatting?

**Comment by pradyunsg** [comment]
If it's malicious, I think contacting as per PyPI's Security Policy is what you'd want to do: https://pypi.org/security/

**Comment by asottile** [comment]
`security at python dot org` seems a bit high of an escalation for something like this, my case is similar to #4413

**Comment by asottile** [comment]
Makes sense, in this case a user scraped github and registered many names of almost-ready-to-publish packages and then was extorting the repository owners.  Whether malicious is the right word there is up to interpretation heh :)

**Comment by di** [comment]
@asottile Ah, that's something else entirely. Could you send the details to me? di@python.org.

**Comment by asottile** [comment]
Awesome, shot you a quick email -- thanks for the resolution here :)
